uwsgi: Add user/group for uwsgi service.

oconnorr opened the pull request:
Also add a uwsgi directory under /run (defaulting to /run/uwsgi) where the uwsgi user can place sockets.

oconnorr commented on nixos/modules/services/web-servers/uwsgi.nix:
```diff
@@ -89,24 +95,47 @@ in {
 
-  config = mkIf cfg.enable {
-
+  config = mkIf cfg.enable (let
```
Remove useless let

lethalman commented on nixos/modules/services/web-servers/uwsgi.nix:
```diff
@@ -89,24 +95,46 @@ in {
+      user = mkOption {
+        type = types.str;
+        default = "uwsgi";
+        description = "User account under which nginx runs.";
```
Not necessarily nginx, say web server or whatelse.

oconnorr replied:
Oops.  Thanks for catching this.

oconnorr committed:
uwsgi-service: Add user/group for uwsgi service.
Also add a uwsgi directory under /run (defaulting to /run/uwsgi) where the uwsgi user can place sockets.

lethalman commented:
Looks good to me.
